src/logplex_logging.erl

```diff
@@ -5,7 +5,15 @@
 %% @end
 -module(logplex_logging).
 
--export([dest/2]).
+-export([dest/2
+        ,setup/0]).
+
+-include("logplex_logging.hrl").
+
+-define(SYSLOG_TAB, syslog_tab).
+-define(SYSLOG_HOST, {127, 0, 0, 1}).
+-define(SYSLOG_PORT, 514).
+-define(SYSLOG_FACILITY, local2).
```

Comment:
Can we turn these into application variables and create defaults in the `logplex.app.src` file. Or even better make them configurable via env vars.